### ISSUE TITLE
Get 404 /auth/modulus when trying to create a user

### ISSUE THREAD

**Issue by joakim5937x** [emersion/neutron#47]
Trying to create a user but is unable to do so from the /create/new path. Then I press the button on the capcha step it fails 404 because it tries to get /auth/modulus and this path does not exist.

@emersion:
Is there any other way to create a user, for example create the user directly in the database?

Seems like a great project if I could just get it up and running!

Thanks!

**Comment by emersion** [comment]
Yeah, the registration page is broken. I should update instructions.

You don't really need to "create an account". What you need is to put your private and public key in `db/keys`. If your email address is `neutron@example.org`, put your armored public key in `db/keys/example.org/neutron.pub.gpg` and your armored private key in `db/keys/example.org/neutron.priv.gpg`. You can see that these test keys are already there: https://github.com/emersion/neutron/tree/master/db/keys

Once your keys are set up, you can login with your email and your IMAP/SMTP password.
